### ISSUE TITLE
[RFQ] Code-style for cms-bot scripts

### ISSUE THREAD

**Issue by iarspider** [cms-sw/cms-bot#2097]
To make maintaining cms-bot easier, especially for new people, I'd like to propose to start enforcing code format. We already enforce it for main cmssw repo. 

For initial reformat, we can use [black](https://pypi.org/project/black/) - it is guaranteed to preserve AST (abstract syntax tree, i.e. logic). After that, we can add other tools, like [flake8](https://pypi.org/project/flake8/), [mypy](https://pypi.org/project/mypy/)

**Comment by iarspider** [comment]
See https://github.com/cms-sw/cms-bot/pull/2099 for results of reformatting. I will add github actions in a followup commit to that PR.

**Comment by iarspider** [comment]
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#ast-before-and-after-formatting

> To put things in perspective, the code equivalence check is a feature of Black which other formatters don’t implement at all. It is of crucial importance to us to ensure code behaves the way it did before it got reformatted. We treat this as a feature and there are no plans to relax this in the future. The exceptions enumerated above stem from either user feedback or implementation details of the tool. In each case we made due diligence to ensure that the AST divergence is of no practical consequence.

**Comment by smuzaffar** [comment]
thanks @iarspider for the suggestion and setting up the github action to check for `black` format.
I guess this issue resolved and can be closed ... right?

**Comment by iarspider** [comment]
Yes, let's close this, unless we want to implement more stricter checks (via flake8). `black` ensures minimal readability, while tools like `flake8` can do static checks, but the proposed changes are not guaranteed to preserve AST and need to be reviewed/checked.
